src/components/useGetRecipePrice.tsx

```diff
@@ -0,0 +1,73 @@
+import { NewRecipe } from "./types.ts";
+
+type Product = {
+  id: number;
+  preferred?: boolean | undefined;
+  price: {
+    amount: number;
+  };
+  inStock: boolean;
+};
+
+type IngredientData = null | {
+  productsByStoreId: {
+    [storeId: string]: Product[];
+  };
+  ingredientIds: {
+    [storeId: string]: string[];
+  };
+};
+
+export const useGetRecipePrice = (
+  ingredientData: IngredientData,
+  recipe: NewRecipe,
+) => {
+  let totalPrice = 0;
+  const productIds: number[] = [];
+  try {
+    if (ingredientData != null && recipe) {
```

Comment:
Tahle funkce se mi libi, jen bych si primo tady natahnul ty data pokud je potrebujem 👍

Reply:
mam je natahnout primo tady i pokud je pak budu pouzivat v te recipecomponent? nebo pokud je tam vyuziju, tak to mam nechat v props?